libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -1,9 +1,36 @@
 <ng-container *ngIf="(readingList$ | async).length > 0; else empty">
   <div class="reading-list-item" *ngFor="let b of readingList$ | async">
+    <div *ngIf="b.finished" class="finished">
+      <button
+        mat-icon-button
+        color="primary"
+        [attr.aria-label]="'Mark ' + b.title + ' not yet finished'"
+        (click)="markBookNotFinished(b)"
+      >
+        <mat-icon
+          matTooltip="You finished this book on {{ b.finishedDate | date:'MM/dd/yy' }}. Click to mark it as not finished.">
```

Comment:
UX-wise it would be preferable if this text was actually legible in page somewhere. Two-digit dates are also a problem... Ideally we would use ISO-8601